src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -181,6 +183,11 @@ private void handleExit() {
         primaryStage.hide();
     }
 
+    private void handleCopy(CommandResult result) {
+        commandBox.enableButton();
+        commandBox.setCopiedText(result.getFeedbackToUser());
```

Comment:
I am not entirely sure if it is possible, but why not just do `clipboard.setContent` here so the user won't have to press `copyButton` in order to copy the command? That way, it makes it more typing-centric. You can output a `Successfully copied `PersonField` of person at `INDEX` to indicate to user that it has been copied.